winml/test/api/LearningModelSessionAPITest.cpp

```diff
@@ -311,6 +311,37 @@ static void EvaluateSessionAndCloseModel()
     WINML_EXPECT_NO_THROW(::EvaluateSessionAndCloseModelHelper(LearningModelDeviceKind::Cpu, false));
 }
 
+static void NamedDimensionOverride() 
+{
+  LearningModel model = nullptr;
+  WINML_EXPECT_NO_THROW(APITest::LoadModel(L"fns-candy.onnx", model));
+
+  LearningModelDevice device(nullptr);
+  WINML_EXPECT_NO_THROW(device = LearningModelDevice(LearningModelDeviceKind::Cpu));
+
+  LearningModelSessionOptions options;
+  options.OverrideNamedDimension(L"None", 5);
+  
+  // Verifies that if a Dim name doesn't exist the named dimension override does nothing
+  options.OverrideNamedDimension(L"DimNameThatDoesntExist", 10);
+
+  LearningModelSession session(nullptr);
+  WINML_EXPECT_NO_THROW(session = LearningModelSession(model, device, options));
+
+  ILearningModelFeatureDescriptor descriptor = model.InputFeatures().GetAt(0);
+  TensorFeatureDescriptor tensorDescriptor = nullptr;
+  descriptor.as(tensorDescriptor);
+  std::vector<int64_t> shape{5,3,720,720};
```

Comment:
preference is to name these constants as static_const and then something like N, C, H, and W. Especially because they are used elsewhere, like right below to calculate the size, and earlier to set the named dim override.

Reply:
Good point!

Reply:
fixed